tango/workspace.py

```diff
@@ -70,6 +71,16 @@ def from_json_dict(cls, json_dict: Dict[str, Any]) -> "Run":
         return cls.from_params(params)
 
 
+class WorkspaceSort(StrEnum):
```

Comment:
This is `RunSort`, no?

Reply:
Or maybe these should just be called `Field`, since you could use the same names for filtering?

Reply:
I think `RunSort` makes sense. b5e7111